example/index.html

```diff
@@ -1,29 +1,53 @@
-<!doctype html>
+<!DOCTYPE html>
```

Comment:
I've nuked this in my PR. We should touchbase when you're next on

Reply:
I am injecting the PAC configuration here https://github.com/OHIF/Viewers/blob/70ddba7aedc034db2a67a2dac0ec199f9eb02556/example/index.html#L83

generated from docker-compose like:
https://github.com/OHIF/Viewers/blob/70ddba7aedc034db2a67a2dac0ec199f9eb02556/docker/docker-compose-orthanc.yml#L27